usersession/agent/rest_api.go

```diff
@@ -323,6 +323,8 @@ func postRefreshFinishedNotification(c *Command, r *http.Request) Response {
 		})
```

Comment:
this is invoked by the snap run code as well, via the code in /cmd/snap/inhibit.go that might also not work/be relevant, we should probably remove some of that code if we do this from snapd instead?

Reply:
In fact, I'm trying to send a notification saying that the snap has been fully updated from that function, but it seems to not work...

Reply:
I am happy to cleanup the "snap run" codepath for this, I would like to do it in a followup to keep churn minimal but I looked and I think the inhibit ocde in snap run is really not working and users will never see that anyway :(

Reply:
I cleaned up the snap run code now, it will not send notifications (those were pretty broken anyway) and double checked with Oliver what flow he expects.